bloom-housing/bloom
# 1382/listings management, application dates

# Pull Request Template

## Issue

Addresses #1382 

## Description

Adds the application dates section to the listings management MVP. Note: our existing time fields has a default `seconds` input which I don't think we always need, but moving forward with it as is for now.

## Type of change


## How Can This Be Tested/Reviewed?

Create a new listing from partners, check out the new section at the bottom, save the listing and checkout the detail page, re-edit the listing and ensure the data is properly loaded. Preview the listing and see the new time field at the top of the right hand panel.

## Checklist:
